App.js

```diff
@@ -138,9 +138,7 @@ const App = () => {
             linking={linking}>
             <View>
               <OverlayProvider topInset={top} bottomInset={bottom} i18nInstance={streami18n}>
-                <KeyboardAvoidingView enabled={Platform.OS === 'ios'} behavior="padding">
-                  <RootNavigator areaHeight={height} currentScreen={currentScreen} />
-                </KeyboardAvoidingView>
+                <RootNavigator areaHeight={height} currentScreen={currentScreen} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `KeyboardAvoidingView` component has been removed from this file and moved to `root-stack.js`. This change seems to be part of a refactoring effort to improve the organization of the code.
```diff
-                <KeyboardAvoidingView enabled={Platform.OS === 'ios'} behavior="padding">
-                  <RootNavigator areaHeight={height} currentScreen={currentScreen} />
-                </KeyboardAvoidingView>
+                 <RootNavigator areaHeight={height} currentScreen={currentScreen} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->